CONTRIBUTING.md

```diff
@@ -1,6 +1,24 @@
 # Contributing
 
+## Adding plugins
+
+When adding or upgrading a plugin, we should see changes in both `package.json`
+and `yarn.lock`.
+
+You'll need to [install yarn](https://yarnpkg.com/en/docs/install).
```

Comment:
Ah, right. I was just removing the `shrinkwrap` task from the Makefile, but I should be able to adapt it instead, right?